.github/actions/launchable/setup/action.yml

```
@@ -264,10 +264,11 @@ runs:
         shell: bash
         working-directory: ${{ inputs.srcdir }}
         post: |
-          [[ "${test_all_enabled}" = "true" ]] && \
+          if [[ "${test_all_enabled}" == "true" ]]; then \
```

This escape is not needed too.
```suggestion
          if [[ "${test_all_enabled}" == "true" ]]; then
```